torch/_inductor/codegen/cpp.py

```diff
@@ -398,6 +408,43 @@ def expm1(x):
     def log1p(x):
         return f"{x}.log1p()"
 
+    @staticmethod
+    def masked(mask, body, other):
+        assert "is_masked_load" in V.interpreter.current_node.meta
+        assert V.interpreter.current_node.meta["is_masked_load"]
+        code = BracesBuffer()
+
+        var = V.kernel.cse.newvar()
+        if other == float("-inf"):
+            code.writeline(
+                f"auto {var} = at::vec::Vectorized<float>(-std::numeric_limits<float>::infinity());"
+            )
+        elif other == float("inf"):
+            code.writeline(
+                f"auto {var} = at::vec::Vectorized<float>(std::numeric_limits<float>::infinity());"
+            )
+        elif isinstance(other, float):
+            code.writeline(f"auto {var} = at::vec::Vectorized<float>({other});")
```

Comment:
Won't the next case do the same thing for a float?

Reply:
Fixed by removing this logic.